### ISSUE TITLE
Erroneous `cargo-features = ["edition2024"]` requirement in projects using `nightly` when this not needed any more

### ISSUE THREAD

**Issue by virtualritz** [est31/cargo-udeps#292]
See subject -- [`edition = "2024"`](https://doc.rust-lang.org/edition-guide/rust-2024/index.html) was stabilized on `nightly` on Nov. 28th. If one uses a `nightly` after that (I do) `cargo` notices the user that the resp. line `cargo-features = ["edition2024"]` can be removed from `Cargo.toml` (I did). 

However, `cargo udeps` (and `cargo +nightly udeps`) refuse to run even when the resp. project is using `nightly` and the locally installed `nighly` toolchain is current (i.e. after Nov. 28th).

As this affects all local dependencies this is not just adding a single line to make the error go away but requires (re-)adding this to _all_ local depdencies' `Cargo.toml`'s that have `edition = "2024"` set (about two dozen, in my case).

Example:

```
error: failed to get `openimageio-sys` as a dependency of package `openimageio v0.1.0 (/home/ritz/code/crates/openimageio)`
    ... which satisfies path dependency `openimageio` (locked to 0.1.0) of package `nci-cli v0.1.0 (/home/ritz/code/crates/nci/crates/nci-cli)`

Caused by:
  failed to load source for dependency `openimageio-sys`

Caused by:
  Unable to update /home/ritz/code/crates/openimageio/openimageio-sys

Caused by:
  failed to parse manifest at `/home/ritz/code/crates/openimageio/openimageio-sys/Cargo.toml`

Caused by:
  feature `edition2024` is required

  The package requires the Cargo feature called `edition2024`, but that feature is not stabilized in this version of Cargo (1.80.0).
  Consider adding `cargo-features = ["edition2024"]` to the top of Cargo.toml (above the [package] table) to tell Cargo you are opting in to use this unstable feature.
  See https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#edition-2024 for more information about the status of this feature.
```

**Comment by est31** [comment]
Hi, thanks for beta testing the 2024 edition.

As described in my comment here, you can use a local build of cargo-udeps until the 2024 edition hits stable: https://github.com/est31/cargo-udeps/issues/290#issuecomment-2518321252

**Comment by est31** [comment]
For anyone subscribed, now that edition 2024 is on stable, I've updated cargo and released [0.1.55](https://github.com/est31/cargo-udeps/releases/tag/v0.1.55) which supports edition 2024.
